crates/js-sys/src/lib.rs

```diff
@@ -4742,3 +4787,70 @@ pub fn global() -> Object {
 
     GLOBAL.with(|g| g.clone())
 }
+
+macro_rules! arrays {
+    ($($name:ident: $ty:ident,)*) => ($(
+        impl $name {
+            /// Creates a JS typed array which is a few into wasm's linear
```

Comment:
Just a minor thing that I noticed -> `few` -> `view`

Reply:
Oops, I'll be sure to handle before landing!